portfolio/src/main/webapp/index.html

```diff
@@ -135,20 +140,25 @@ <h3>Coursera Machine Learning Course</h3>
         <img src="img/bot.png" class="blogimg"/>
        
         </div>
-         <a href="blogs/blog0.html" <button type="button" class="btn btn-sm btn-outline-secondary">Read More</button></a>
+         <a href="blogs/blog0.html"><button type="button" class="btn btn-sm btn-outline-secondary">Read More</button></a>
     </div>
     
 
    
 
     <div class="blankspace"></div> <!-- necessary to allow for scrolling to blog section" -->
 
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"></script>
+  <script src="https://pingendo.com/assets/bootstrap/bootstrap-4.0.0-alpha.6.min.js"></script>
 
     </body>
+   
+    <footer>
```

Comment:
All page content that you want rendered needs to go in `<body>`. 

Note that `head` and `body` refer to parts of the `html` that are distinguished based on how the browser treats them; `head` is generally for non-rendered things like metadata, while `body` is for everything you want to be displayed.

The counterpart to `footer` is `header` which is different from the `head` tag.

`header`, `footer`, and `main` are part of HTML5's semantic elements, like `section`, and `article`. These tags don't do very much on their own, and mostly just act like `div`s, but exist so you can lay out your page semantically, by tagging things with what they are intended to mean. They all have to go in the `body` and there are some extra rules for where they are supposed to appear, e.g. you're not supposed to put a `header` inside another `header` or `footer`, but either way they mostly just act like divs that can be styled independently without needing to use a special class for them.

Reply:
I also moved `header` within `body`

Reply:
This is still below the `</body>` end tag.